frontend/shadow-cljs.edn

```diff
@@ -29,19 +29,21 @@
                                    imas-seamap.specs.preload]}
 
           :min {:target            :browser
-                :closure-defines   {goog.DEBUG                  false
-                                    imas-seamap.db/api-url-base "https://data.imas.utas.edu.au/seamap/api/"
-                                    imas-seamap.db/img-url-base "/app/img/"}
+                :closure-defines   {goog.DEBUG                    false
+                                    imas-seamap.db/api-url-base   "https://data.imas.utas.edu.au/seamap/api/"
+                                    imas-seamap.db/media-url-base "https://data.imas.utas.edu.au/seamap/media/"
```

Comment:
Aah, yeah I'm not sure about this.  It would be nice if we could avoid that here; worth a thought (I guess the frontend needs the url base... hmm)

Reply:
There's a few hacky values like that now, I wonder if it's worth injecting in an initial bootstrap payload... not the most important thing, it's just looking ugly.